definitions/npm/moment_v2.x.x/flow_v0.25.x-/moment_v2.x.x.js

```diff
@@ -169,11 +169,11 @@ declare class moment$Moment {
   days(day: number | string): this;
   day(): number;
   days(): number;
-  weekday(day: number | string): this;
+  weekday(number: number): this;
   weekday(): number;
   isoWeekday(day: number | string): this;
```

Comment:
This was incorrect in v2.3.x version, but correct in the v2.x.x version:

https://github.com/flow-typed/flow-typed/blob/78c72eef4472905c8248ceeb51842dc912c5435a/definitions/npm/moment_v2.3.x/flow_v0.25.x-/moment_v2.3.x.js#L189